spel/scripts/amigen7-build.sh

```diff
@@ -220,6 +219,78 @@ function CollectManifest {
     rpm --root "${CHROOT}" -qa | sort -u >> /tmp/manifest.txt
 }
 
+function ComposeAWSutilsString {
+    # Construct the cli option string for aws utils
+    CLIOPT_AWSUTILS=("-m ${CHROOT}" "-d ${ELBUILD}/AWSpkgs")
+
+    # Whether to install AWS CLIv1
+    if [[ -n "${AWSCLIV1SOURCE}" ]]
+    then
+      CLIOPT_AWSUTILS+=("-C ${AWSCLIV1SOURCE}")
+    fi
+
+    # Whether to install AWS CLIv2
+    if [[ -n "${AWSCLIV2SOURCE}" ]]
+    then
+      CLIOPT_AWSUTILS+=("-c ${AWSCLIV2SOURCE}")
+    fi
+}
+
+function ComposeCliOptionString {
+    # Construct the cli option string for alternate-manifest
+    CLIOPT_ALTMANIFEST=""
+    if [[ -n ${AMIGENMANFST} ]]
+    then
+       CLIOPT_ALTMANIFEST=( -m "${AMIGENMANFST}" )
+       echo "Sending manifest-option '${CLIOPT_ALTMANIFEST[*]}'"
+    else
+       CLIOPT_ALTMANIFEST=( -g "${AMIGENPKGGRP}" )
+       echo "Sending manifest-option '${CLIOPT_ALTMANIFEST[*]}'"
+    fi
+    
+    
+    # Construct the cli option string for extra rpms
+    CLIOPT_EXTRARPMS=""
+    if [[ -n "${EXTRARPMS}" ]]
+    then
+        CLIOPT_EXTRARPMS=(-e "${EXTRARPMS}")
+    fi
+    
+    # Construct the cli option string for a custom repo
+    CLIOPT_CUSTOMREPO=""
+    if [[ -n "${CUSTOMREPORPM}" && -n "${CUSTOMREPONAME}" ]]
+    then
+        CLIOPT_CUSTOMREPO=(-r "${CUSTOMREPORPM}" -b "${CUSTOMREPONAME}")
+    fi
+}
+
+function ComposeCliOptionString {
```

Comment:
This function is now present twice, causing shellcheck to freak out (technical term)